exec-service-wasmer/src/wasmer_executor.rs

```diff
@@ -10,24 +10,53 @@ pub struct WasmerExecutor {
     pub data: Rc<WasmerExecutorData>,
 }
 
+#[derive(PartialEq, PartialOrd, Eq)]
+pub enum WasmerExecutorLogLevel {
+    None,
+    Debug,
+    Trace,
+}
+
+impl TryFrom<u64> for WasmerExecutorLogLevel {
+    type Error = &'static str;
+
+    fn try_from(value: u64) -> Result<Self, Self::Error> {
```

Comment:
Implementation feels overly complicated. For maximum readability, I would simply do
```
#[derive(PartialEq, PartialOrd, Eq)]
pub enum WasmerExecutorLogLevel {
    None = 0,
    Debug = 1,
    Trace = 2,
}

impl TryFrom<u64> for WasmerExecutorLogLevel {
    type Error = &'static str;

    fn try_from(value: u64) -> Result<Self, Self::Error> {
        match value {
            0 => Ok(WasmerExecutorLogLevel::None),
            1 => Ok(WasmerExecutorLogLevel::Debug),
            2 => Ok(WasmerExecutorLogLevel::Trace),
            _ => Err("WasmerExecutor undefined log level"),
        }
    }
}
```
?

Reply:
Done.